src/devices/Arduino/ArduinoAnalogController.cs

```diff
@@ -81,7 +81,7 @@ protected override AnalogInputPin OpenPinCore(int pinNumber)
             // This method is called with the logical pin numbering (input pin A0 is 0, A1 is 1, etc)
             // but the SetPinMode method operates on the global numbers
             int fullNumber = ConvertLogicalNumberingSchemeToPinNumber(pinNumber);
-            _board.Firmata.SetPinMode(fullNumber, SupportedMode.AnalogInput);
+            _board.Firmata.SetPinMode(fullNumber, SupportedMode.AnalogInput.Value);
```

Comment:
I'm confused about this - why is SetPinMode taking a byte rather than PinMode/SupportedMode?

Reply:
This is the internal (low-level) value. But maybe I should move that conversion inside the function.